Bottom: 0259758746426b71cd983a2073616ca4d4a81135
Top:    eaa41893158110a4df8aee7cd92bf1ab4c611d9d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 23:11:25 +0000

Refresh of hid-gerber-remove-dead-colour-

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index c0faf47..18de0f7 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -305,7 +305,6 @@ static int pagecount = 0;
 static int linewidth = -1;
 static int lastgroup = -1;
 static int lastcap = -1;
-static int lastcolor = -1;
 static int print_group[MAX_LAYER];
 static int print_layer[MAX_LAYER];
 static int lastX, lastY;	/* the last X and Y coordinate */
@@ -589,7 +588,6 @@ gerber_do_export (HID_Attr_Val * options)
   linewidth = -1;
   lastcap = -1;
   lastgroup = -1;
-  lastcolor = -1;
 
   region.X1 = 0;
   region.Y1 = 0;
@@ -705,7 +703,6 @@ gerber_set_layer (const char *name, int group, int empty)
       lastgroup = group;
       lastX = -1;
       lastY = -1;
-      lastcolor = 0;
       linewidth = -1;
       lastcap = -1;
 
@@ -958,30 +955,6 @@ use_gc (hidGC gc, int radius)
       if (f)
 	fprintf (f, "G54D%d*", aptr->dCode);
     }
-#if 0
-  if (lastcolor != gc->color)
-    {
-      c = gc->color;
-      if (is_drill)
-	return;
-      if (is_mask)
-	c = (gc->erase ? 0 : 1);
-      lastcolor = gc->color;
-      if (f)
-	{
-	  if (c)
-	    {
-	      fprintf (f, "%%LN%s_C%d*%%\r\n", layername, lncount++);
-	      fprintf (f, "%%LPC*%%\r\n");
-	    }
-	  else
-	    {
-	      fprintf (f, "%%LN%s_D%d*%%\r\n", layername, lncount++);
-	      fprintf (f, "%%LPD*%%\r\n");
-	    }
-	}
-    }
-#endif
 }
 
 static void
